join.spec.md

```diff
@@ -100,204 +101,347 @@ flowchart TB
 
-# Basic Requirements
+# Data Model
 
-Schemas using {@join} MUST be valid [core schema documents](/core/v0.1) with {@core} directives referencing this specification.
+TODO maybe this should be called something more like "Overview" or "Graph Routing in a Supergraph" or something?
 
-:::[example](photos.graphql#schema) -- {@core} directives for supergraphs
+*This section is non-normative.* It describes the motivation behind the directives defined by this specification.
 
-# Enums
+A supergraph schema describes a GraphQL schema that can be served by a router. The router does not contain logic to resolve any of the schema's fields; instead, the supergraph schema contain directives starting with {@join__} which tell the router which subgraph endpoints can resolve each field, as well as other information needed in order to construct subgraph queries.  The directives described in this specification are designed for a particular query planning algorithm, and so there are some restrictions on how they can be combined that originate from the requirements of this algorithm. (We hope that future versions of this specification can relax some of these restrictions.)
 
-##! join__Graph
+Each supergraph schema contains a list of the subgraphs. It represents this as an enum called [{join__Graph}](#join__Graph) with an enum value for each subgraph. Each enum value is annotated with a [{@join__graph}](#@join__graph) directive telling the router what endpoint can be used to reach the subgraph and a name for the subgraph that can be used in representations of query plans and diagnostic messages.
 
-Enumerate subgraphs.
+To resolve a field, the router needs to know to which subgraphs it can delegate the field's resolution. The most direct way to indicate this in a supergraph schema is by annotating the field with a [{@join__field}](#@join__field) directive specifying which subgraph should be used to resolve that field.
 
-```graphql definition
-enum join__Graph
+In order for the router to send an operation that resolves a given field on a parent object to a subgraph, the operation needs to first resolve the parent object itself. There are several ways to accomplish this, described below. The examples below include abbreviated versions of the supergraph schemas which do not include the `schema` definition, directive definitions, or the `join__Graph` definition. This specification does not require the subgraph queries to be the same as those described in these examples; this is just intended to broadly describe the meanings of the directives.
+
+## Root fields
+
+If this field appears at the root of the overall operation (query or mutation), then the field can just be placed at the root of the subgraph operation.
```

Comment:
"this" field reads a little funny - which field?
```suggestion
If a field appears at the root of the overall operation (query or mutation), then it can be placed at the root of the subgraph operation.
```

Reply:
Applied. Now I'll go to the same to the other sections.

Reply:
aab84d4